texar/torch/modules/embedders/embedder_base.py

```diff
@@ -141,9 +141,5 @@ def forward(self,  # type: ignore
         return input_tensor * mask
 
     @property
-    def output_size(self) -> int:
-        r"""The feature size of :meth:`forward` output.
-        Equals to ``1`` because here output size equals
-        to the input size.
-        """
-        return 1
+    def output_size(self):
+        raise ValueError("'output_size' can not be calculated.")
```

Comment:
Let's also give an explanation, saying that `output_size` cannot be calculated because it is equal to the input size.